Makefile.am

```diff
@@ -369,7 +369,9 @@ endif
 
 check-best-practices:
 if USEPYTHON
-	$(PYTHON) $(top_srcdir)/scripts/maint/practracker/practracker.py $(top_srcdir)
+	@if test "$$TOR_DISABLE_PRACTRACKER" = ""; then \
```

Comment:
Can we implement this check in the practracker script instead?

The git hooks call practracker directly, not through the Makefile.

Reply:
Sure.  Fixed in 3221dc1b32f651